bq-connector/setup/doc_reference_table_schema.json

```diff
@@ -46,6 +46,11 @@
         "type": "DATETIME",
         "mode": "REQUIRED"
     },
+    {
+        "name": "raw_entities",
```

Comment:
I don't think raw_entities is saving into the doc_reference table, I thought it just saves to the destination table?

Reply:
Indeed, was not meant to be committed. Good catch.

Reply:
Question - there is not sample schema for the "destination" table, or explaining what it should be. As desired?

Reply:
It's certainly a good idea to add an example. If we were to add an example schema, I think it would be helpful for an example document, etc.